src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.html

```diff
@@ -1,6 +1,18 @@
 <!-- Individual Student's Page -->
 <div *ngIf="!isFinished() && entries?.length > 0">
-  <app-child-block [entity]="entries[currentIndex].child"></app-child-block>
+  <!--  <app-child-block [entity]="entries[currentIndex].child"></app-child-block>-->
+  <app-child-block
+    [entity]="entries[currentIndex].child"
+    *ngFor="let entry of data?.parts"
```

Comment:
I think `*ngFor` is not a good approach here for it will display all the parts at the same time and not one after the other